### PR TITLE
Made some minor changes to sdk-metric-parameter-tracking-for-locally-trained-models

### PR DESCRIPTION
**Pull request by Tharun-SpringML**
Made some minor changes to sdk-metric-parameter-tracking-for-locally-trained-models
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:8]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:8]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (andrewferlitsch) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:8]: <p>REGION should not be hardwired</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (Tharun-SpringML) [reply notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:8]: Changed to Vertex AI Workbench

Comment (Tharun-SpringML) [reply notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:8]: Changed to Vertex AI Workbench

Comment (Tharun-SpringML) [reply notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:8]: Edited. Defined REGION variable inside an if statement and not hardcoded.

**Review by andrewferlitsch — APPROVED** [review]

**Comment by andrewferlitsch** [comment]
This PR test is running multiple notebooks, instead of just the one:

build_tag                                                             status    duration    log_url
Step #4: --------------------------------------------------------------------  --------  ----------  --------------------------------------------------------------------------------------------------------------
Step #4: sdk-metric-parameter-tracking-for-locally-trained-models.ipynb        PASSED    00:04:24    https://console.cloud.google.com/cloud-build/builds/4e3fde53-e599-4325-aab2-3653f2b6dee3?project=1012616486416
Step #4: google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb  PASSED    00:42:28    https://console.cloud.google.com/cloud-build/builds/0c4a5322-5c83-4e79-96fe-c750a7e6ab0e?project=1012616486416
Step #4: tabnet_vertex_tutorial.ipynb                                          PASSED    00:18:58    https://console.cloud.google.com/cloud-build/builds/824d8293-9a60-4334-941f-fa09874d0c38?project=1012616486416
Step #4: get_started_with_hpt_pipeline_components.ipynb                        FAILED    00:03:45    https://console.cloud.google.com/cloud-build/builds/b2857687-7178-4a39-8262-c80804e3b78b?project=1012616486416
Step #4: get_started_with_rapid_prototyping_bqml_automl.ipynb                  FAILED    00:04:57    https://console.cloud.google.com/cloud-build/builds/270693ad-4af1-4444-b09a-cf7ffcbd0ffb?project=1012616486416
Step #4: custom-tabular-bq-managed-dataset.ipynb                               FAILED    00:03:18    https://console.cloud.google.com/cloud-build/builds/1d2d248e-1721-462d-b2f9-bb19ae97d1f0?project=1012616486416
Step #4:

**Comment by Tharun-SpringML** [comment]
Executed the notebook on workbench and colab. It ran successfully on both.

**Comment by andrewferlitsch** [comment]
automated tests not running. Submitter manually verified ran in Colab and Workbench. I am manually merging PR.

**Pull request merged**